includes/documents/abstract-wcpdf-order-document.php

```diff
@@ -757,6 +794,26 @@ public function shop_name() {
 		echo $this->get_shop_name();
 	}
 	
+	/**
+	 * Return/Show company VAT number
+	 */
+	public function get_shop_vat_number() {
+		return $this->get_settings_text( 'vat_number', '', false );
+	}
+	public function shop_vat_number() {
```

Comment:
I thought about that some time ago, I think I discuss that with Ewout, the previous owner of the company, and his idea was to avoid having the `echo` in the PDF templates to simplify the code there. I think we should keep like that for now.